src/renderer/Components/MyPackages/index.jsx

```diff
@@ -134,13 +135,36 @@ function MyPackages() {
                       {` "${order.sourcePath}" 로`}
                     </span>
                   )}
+                  <span className="font-bold text-blue-700">
+                    {` ${order.action}`}
+                  </span>
                 </div>
               ))}
             </div>
-            <div className="mt-6 flex justify-between text-right">
-              {/* TODO: 추후에 링크 기능 추가 */}
-              <p className="text-sm text-gray-400">
-                일련번호: {userPackage.serialNumber}
+            <div className="mt-6 flex flex-col justify-between text-left">
+              <p className="mb-2 text-sm text-gray-500">
+                <span className="font-bold">링크: </span>
+                <span
+                  className="cursor-pointer hover:font-bold"
+                  onClick={() => {
+                    navigator.clipboard.writeText(
+                      `electron-deliorder://open?packageId=${userPackage.serialNumber}`,
+                    );
+                  }}
+                >
+                  {`electron-deliorder://open?packageId=${userPackage.serialNumber}`}
```

Comment:
아래 리뷰 남겨듯이 여기서도 해당 함수와 변수를 재사용 할 수 있을것 같습니다.😁
해당 변수는 deepLinkURL 는 자주 사용되는 것 같으니 일반변수말고 환경변수로 사용해도 괜찮을 것 같습니다.😄
```suggestion
                  onClick={() => copyToClipboard(deepLinkURL)}
                >
                  {deepLinkURL}
```

Reply:
deepLinkURL을 만드는 함수를 유틸함수화 처리 해보고자합니다🤓👍